packages/project/src/validation.ts

```diff
@@ -74,7 +74,7 @@ const validationRules = {
  * @example <caption>Validating project data using data that has an invalid number of fields defined:</caption>
  *
  * ```js
- * validateProjectData({version:'4.0.6', data}); // returns {valid:false, ...errors}
+ * validateProjectData(data); // returns {valid:false, ...errors}
```

Comment:
I also updated this documentation, since `validateProjectData` doesn't actually take a version string

Reply:
Thanks!